test/test_math.hpp

```diff
@@ -86,11 +87,43 @@ PTESTfix_2(exponentiation_by_squaring_v, uint32_t, 9);
 PTESTfix_2(exponentiation_by_squaring_v, int32_t, 0);
 PTESTfix_2(exponentiation_by_squaring_v, int32_t, 4);
 PTESTfix_2(exponentiation_by_squaring_v, int32_t, 9);
-// PTESTfix_2(exponentiation_by_squaring_v, float, 0);
+PTESTfix_2(exponentiation_by_squaring_v, float, 0);
 PTESTfix_2(exponentiation_by_squaring_v, float, 4);
 PTESTfix_2(exponentiation_by_squaring_v, float, 9);
-// PTESTfix_2(exponentiation_by_squaring_v, double, 0);
+PTESTfix_2(exponentiation_by_squaring_v, double, 0);
 PTESTfix_2(exponentiation_by_squaring_v, double, 4);
 PTESTfix_2(exponentiation_by_squaring_v, double, 9);
+
+LTEST(exp_f32, {
+    IReg dest, src, n;
+    USE_CONTEXT_(CTX);
+    STARTFUNC_(TESTNAME, &dest, &src, &n)
+    {
+        auto expc = expInitConsts(CTX);
+        IReg offset = CONST_(0);
+        WHILE_(n > 0)
+        {
+            VReg<float> x = loadvx<float>(src,offset);
+            storevx(dest, offset, exp(x, expc));
+            offset += CTX.vectorRegisterSize();
```

Comment:
Need to shorter name for 'vectorRegisterSize()'

Reply:
Done.